conmon/conmon.c

```diff
@@ -1072,13 +1073,17 @@ static gboolean terminal_accept_cb(int fd, G_GNUC_UNUSED GIOCondition condition,
 	free(console.name);
 
 	/* We change the terminal settings to match kube settings */
-	if (tcgetattr(console.fd, &tset) == -1)
-		pexit("Failed to get console terminal settings");
+	if (tcgetattr(console.fd, &tset) == -1) {
+		console_failed = TRUE;
```

Comment:
should we instead `return G_SOURCE_CONTINUE;` as we do for the previous error?

Reply:
/cc @mrunalp

Reply:
@giuseppe Yes, I think we should.

Reply:
@mrunalp pushed a new version